oak_containers_kernel/configs/6.1.33/minimal.config

```diff
@@ -11,7 +11,6 @@ CONFIG_AS_VERSION=24000
 CONFIG_LD_IS_BFD=y
 CONFIG_LD_VERSION=24000
 CONFIG_LLD_VERSION=0
-CONFIG_RUST_IS_AVAILABLE=y
```

Comment:
Why is this change needed?

Reply:
I don't know how it appeared. Probably after formatting. Will try putting it back.

Reply:
Done